auth-web/src/components/auth/staff/continuation-application/ContinuationApplicationTable.vue

```diff
@@ -549,6 +602,10 @@ export default defineComponent({
     padding-left: 3px !important;
     padding-right: 3px !important;
   }
+}
 
+// The date picker position
+.date-picker {
+  left: 40%;
```

Comment:
Move the date picker to the right

Reply:
If you don't do this, is is centered? Ask Janis if centered is ok.

Reply:
Yes it's centered, I will confirm with Janis
![image](https://github.com/user-attachments/assets/cff6d11a-6641-4686-bef3-3f3faa125259)